dev-util/edb-debugger/edb-debugger-1.0.0.ebuild

```diff
@@ -19,13 +19,14 @@ S="${WORKDIR}/${P}"
 RDEPEND="
 	dev-libs/capstone
 	graphviz? ( media-gfx/graphviz )
```

Comment:
Please use the opportunity to properly sort these lines (optional deps below unconditional)

Reply:
Will do tomorrow unfortunately, no more time now. I am afraid I did not quite understand what you want me to do with "put optional deps below unconditional".

Thank you.